### PR TITLE
Make env match network when server start And default local network use fixed directory

### PR DESCRIPTION
**Pull request by baichuan3**
resolve https://github.com/rooch-network/rooch/issues/857 and resolve https://github.com/rooch-network/rooch/issues/866

## Summary

1. Automatically switch env when use start server, if env is local or dev seed
2. Verify env alias uniqueness
3. The default local network use fixed directory, modify the test case to support temporary directory
4. Remove wallet RPC

### CODE REVIEW

File: crates/rooch-rpc-client/src/client_config.rs [hunk 1]
```diff
@@ -15,8 +15,8 @@ use serde_with::serde_as;
 use std::fmt::{Display, Formatter, Write};
 
 pub const DEFAULT_EXPIRATION_SECS: u64 = 30;
-pub const ROOCH_DEV_NET_URL: &str = "https://dev-seed.rooch.network/";
-pub const ROOCH_TEST_NET_URL: &str = "https://test-seed.rooch.network/";
+pub const ROOCH_DEV_NET_URL: &str = "https://dev-seed.rooch.network:443/";
```
Comment (feliciss) [inline crates/rooch-rpc-client/src/client_config.rs:18]: The default https URL port is 443? There's no need to add `443`.

File: crates/rooch-rpc-client/src/client_config.rs [hunk 2]
```diff
@@ -15,8 +15,8 @@ use serde_with::serde_as;
...
+pub const ROOCH_TEST_NET_URL: &str = "https://test-seed.rooch.network:443/";
```
Comment (feliciss) [inline crates/rooch-rpc-client/src/client_config.rs:19]: Also there.

**Review by feliciss — APPROVED** [review]
LGTM. Some backend ports may need to be modified to original format without `:port`.

File: crates/rooch-rpc-client/src/client_config.rs (hunk 1 above)
Comment (baichuan3) [reply crates/rooch-rpc-client/src/client_config.rs:18]: It is related to this issue https://github.com/rooch-network/rooch/issues/869. It is caused by a bug in jsonrpsee and needs to be upgraded. I tried the upgrade and found that it is quite troublesome. The upgrade requires the refactor of RPC result and RPC client. This version does not have enough time and will be dealt with in the next version.

**Pull request merged**